accio-base/src/main/java/io/accio/base/wireprotocol/PgMetastore.java

```diff
@@ -12,28 +12,32 @@
  * limitations under the License.
  */
 
-package io.accio.main.wireprotocol;
+package io.accio.base.wireprotocol;
 
-import io.accio.base.Column;
 import io.accio.base.ConnectorRecordIterator;
 import io.accio.base.Parameter;
+import io.accio.base.client.Client;
 import io.accio.base.sql.SqlConverter;
 
 import java.util.List;
 
-public interface PgMetastore
+public abstract class PgMetastore
```

Comment:
Look like PgMetastoreImpl doesn't need to implement multiple interface now. Can we revert this change?